src/DestinationCard/index.js.flow

```diff
@@ -0,0 +1,32 @@
+// @flow
+
+import type { Globals } from "../common/common.js.flow";
+
+export type State = {|
+  hiddenContentHeight: number,
+|};
+
+export type Props = {|
+  ...Globals,
+  +departureCity: string,
```

Comment:
All string props (expect image)should be probably typed with `Translation` type.